view/vocab-shared.twig

```diff
@@ -52,6 +52,9 @@
         {% if 'text/turtle' in vocab.config.dataURLs|keys %}
           <a href="rest/v1/{{ request.vocabid }}/data?format=text/turtle">TURTLE</a>
         {% endif %}
+        {% if 'application/marcxml+xml' in vocab.config.dataURLs|keys %}
+          <a href="rest/v1/{{ request.vocabid }}/data?format=application/marcxml%2Bxml">MARC21/XML</a>
```

Comment:
MARCXML is the standard name for the format. In principle, it could contain another flavor of MARC than MARC21 (e.g. UniMARC, FinMARC or USMARC). So just call it MARCXML, not MARC21/XML